src/App.js

```diff
@@ -1,10 +1,29 @@
 import "./App.css";
-import logo from "./logo.png";
+import Header from "./Components/Header/Header";
+// import Login from './Components/Loginsignup/Login';
+import Login from "./Components/LoginSignUp/Login";
+import {BrowserRouter as Router,Routes,Route } from 'react-router-dom'
+import Product from "./Components/product/product";
```

Comment:
instead of importing components separately, do barrel export, i.e export all the components from an index.js file inside components folder